go/api/base_client.go

```diff
@@ -199,3 +200,107 @@ func (client *baseClient) DecrBy(key string, amount int64) (int64, error) {
 	}
 	return handleLongResponse(result), nil
```

Comment:
Had we using the coding convention, this line could have been:

```go
return handleLongResponse(result)
```

Reply:
@eifrah-aws I have changed it according to the coding convention. I would suggest that we can refactor the coding convention in seperated PR.